mlflow/transformers/__init__.py

```diff
@@ -2552,7 +2552,10 @@ def is_base64_image(image):
         """Check whether input image is a base64 encoded"""
 
         try:
-            return base64.b64encode(base64.b64decode(image)).decode("utf-8") == image
+            return (
+                base64.b64encode(base64.b64decode(image)).decode("utf-8") == image
+                or base64.encodebytes(base64.b64decode(image)).decode("utf-8") == image
```

Comment:
Can we keep `base64.b64decode(image)` as variable and reuse, so we don't need to decode twice?

Reply:
makes sense, fixed.